apps/common/models/i-casework.js

```diff
@@ -15,7 +15,7 @@ module.exports = class CaseworkModel extends Model {
     return config.icasework.url + config.icasework.createpath;
   }
 
-  prepare() {
+  prepare(token) {
```

Comment:
Only in the case of submission will token be defined as it is passed to the prepare step by the Compose function. If we have missed anything for and scenarios where pdf-upload is being grabbed (unlikely), token bearer below will cause a ReferenceError in the application. If we test all journeys and check the unit tests I imagine we should be alright. Again this would be simpler once we refactor out this whole Compose/prepare override gubbins.